projects/GPGame/README.md

```diff
@@ -15,3 +15,19 @@ git submodule update --init --recursive
  - 2 : Run installer and check : add Cmake to system path 
 ![](https://tulip.labri.fr/TulipDrupal/sites/default/files/uploadedFiles/users/6/screenshot_cmake01.png)
 - 3 : Run [installer](install_dependancies.bat) in the project root to download dependancy (git submodule) and install it
```

Comment:
```suggestion
2. Run CMake's installer and check "Add CMake to the system path" (for all/current user)
![](https://tulip.labri.fr/TulipDrupal/sites/default/files/uploadedFiles/users/6/screenshot_cmake01.png)
3. Run the [install_dependancies.bat](install_dependancies.bat) script in the project root to download dependencies (as submodules) and install them
```

I would also rename the script `install_dependancies.bat` to `install_dependencies.bat`. There is currently no such script in your branch.

This README should be at the root of the project.

Line 8, I would also replace :

> Run this command to download all dependencies (this operation can take up to a few minutes)
> ```batch
> git submodule update --init --recursive
> ```

with 

> Start by cloning the project with
> ```
> `git clone --recurse-submodules https://github.com/GP-Engine-team/GP_Engine.git`
> ```
> (this operation can take up to a few minutes)

which will initialize all submodules immediatly